ontology/foodon.md

```diff
@@ -37,6 +37,27 @@ repository: https://github.com/FoodOntology/foodon
 tags:
 - food
 tracker: https://github.com/FoodOntology/foodon/issues/
+usages:
```

Comment:
@ddooley I took the liberty of elaborating the description for most of the usages of FOODON.
Please feel free to reword as appropriate.